app/actions/websocket/teams.ts

```diff
@@ -49,3 +53,43 @@ export async function handleLeaveTeamEvent(serverUrl: string, msg: any) {
         }
     }
 }
+
+export async function handleUpdateTeamEvent(serverUrl: string, msg: WebSocketMessage) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
+        return;
+    }
+
+    try {
+        const team = JSON.parse(msg.data.team) as Team;
+        database.operator.handleTeam({
+            teams: [team],
+            prepareRecordsOnly: false,
+        });
+    } catch {
+        // Do nothing
+    }
+}
+
+export async function handleTeamAddedEvent(serverUrl: string, msg: WebSocketMessage) {
```

Comment:
@enahum Corner case we may want/need to handle.

I belong to no teams, so I am in the "select a team to join" (or "there are no teams to join") screen. I get added to a team. Should we somehow move the user to that new team, or should we stay in that state? Specially the "there are no teams to join" scenario may be specially annoying, since it may imply closing and opening the app again to move forward.

Reply:
Very interesting corner case, in my personal opinion I would say that we know that the user was added and we should take action.

Now here is the thing:
1. We need to ask UX what the action should be
2. We currently do not have a No teams screen (cc: @Rina-dsg @matthewbirtch) this screen would need to allow me not only to join a team if available but should allow me to switch servers as well.
3. We should only do this for the current active server (if we are to do it)

Reply:
@matthewbirtch @Rina-dsg can you help  with the question above?

Reply:
@Willyfrog This is an interesting case. I need to think about the solution. I'll keep you updated.

Reply:
I created a ticket for this corner case and assigned to @Rina-dsg.  https://mattermost.atlassian.net/browse/MM-41404.

We'll address this in a new PR.